### PR TITLE
feat: Update empty screen for sent, received transfers and settings screen

### PR DESCRIPTION
**Pull request by tevincent**
https://github.com/user-attachments/assets/3f87d574-415e-4ea0-9a73-03b20a6b390b

### CODE REVIEW

File: app/src/main/java/com/infomaniak/swisstransfer/ui/components/IllustratedMessageBlock.kt [hunk 1]
```diff
@@ -42,34 +42,40 @@ import com.infomaniak.swisstransfer.ui.theme.SwissTransferTheme
...
 ) {
     Column(
         modifier = modifier,
         horizontalAlignment = Alignment.CenterHorizontally,
     ) {
-        Image(imageVector = icon, contentDescription = null)
-
-        Spacer(Modifier.height(Margin.Huge))
+        icon?.let {
+            Image(imageVector = it, contentDescription = null)
```
Comment (KevinBoulongne) [inline app/src/main/java/com/infomaniak/swisstransfer/ui/components/IllustratedMessageBlock.kt:55]: We lost the Image in ReceivedScreen on mobile.
image

**Review by KevinBoulongne — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/infomaniak/swisstransfer/ui/components/IllustratedMessageBlock.kt (hunk 1 above)
Comment (tevincent) [reply app/src/main/java/com/infomaniak/swisstransfer/ui/components/IllustratedMessageBlock.kt:55]: I've added the image on mobile.

File: app/src/main/java/com/infomaniak/swisstransfer/ui/screen/main/transfers/TransfersScreenWrapper.kt [hunk 2]
```diff
@@ -42,23 +46,31 @@ import kotlinx.parcelize.Parcelize
...
+private fun ListPane(
+    direction: TransferDirection,
+    navigator: ThreePaneScaffoldNavigator<DestinationContent>,
+    hasTransfer: (Boolean) -> Unit
+) {
     when (direction) {
         TransferDirection.SENT -> SentScreen(
             navigateToDetails = { transferUuid -> navigator.navigateToDetails(direction, transferUuid) },
             getSelectedTransferUuid = navigator::getSelectedTransferUuid,
+            hasTransfer = { hasTransfer(it) }
```
Comment (KevinBoulongne) [inline app/src/main/java/com/infomaniak/swisstransfer/ui/screen/main/transfers/TransfersScreenWrapper.kt:68]:
```suggestion
            hasTransfer = hasTransfer,
```

File: app/src/main/java/com/infomaniak/swisstransfer/ui/screen/main/transfers/TransfersScreenWrapper.kt [hunk 3]
```diff
@@ -42,23 +46,31 @@ import kotlinx.parcelize.Parcelize
...
         )
         TransferDirection.RECEIVED -> ReceivedScreen(
             navigateToDetails = { transferUuid -> navigator.navigateToDetails(direction, transferUuid) },
             getSelectedTransferUuid = navigator::getSelectedTransferUuid,
+            hasTransfer = { hasTransfer(it) }
```
Comment (KevinBoulongne) [inline app/src/main/java/com/infomaniak/swisstransfer/ui/screen/main/transfers/TransfersScreenWrapper.kt:73]:
```suggestion
            hasTransfer = hasTransfer,
```

**Review by KevinBoulongne — CHANGES_REQUESTED** [review]

**Review by KevinBoulongne — APPROVED** [review]

**Pull request merged**